lib/iiif_manifest/manifest_builder/record_property_builder.rb

```diff
@@ -19,6 +20,30 @@ def apply(manifest)
       def viewing_hint
         (record.respond_to?(:viewing_hint) && record.send(:viewing_hint))
       end
+
+      # If there is a manifest_metadata method, check that it is an Array containing Hashes
+      #  with pairs of 'label' and value'
+      #
+      # @return Array metadata array of label/value hashes
```

Comment:
methods that end with `?` should return Boolean values. The way to document that in yard is:
```
# @return [Boolean] description of what this means
```